MobileApplication/MobileApplication/Views/AboutPage.xaml.cs

```diff
@@ -124,6 +132,10 @@ void PresentTransactionChart(IEnumerable<Transaction> transactions)
             {
                 Entries = entries,
             };
+            budget.Text = "Income: " + income + "\n" + "Expenses: " + expenses;
+            net = 0;
+            income = 0;
+            expenses = 0;
```

Comment:
Why in the end these values are set to 0?

Reply:
When we represent the data, after that we need to clear the variables in case other data is added